keyboards/kbdfans/kbd19x/kbd19x.h

```diff
@@ -29,74 +29,84 @@ inline void kbd19x_sclk_led_off(void)   { DDRB &= ~(1<<1); PORTB &= ~(1<<1); }
 inline void kbd19x_nmlk_led_on(void)    { DDRB |=  (1<<2); PORTB &= ~(1<<2); }
 inline void kbd19x_nmlk_led_off(void)   { DDRB &= ~(1<<2); PORTB &= ~(1<<2); }
 
-// readability
-#define XXX KC_NO
+#define XXXX KC_NO
 
-/* This is a shortcut to help you visually see your layout.
- *
- * The first section contains all of the arguments representing the physical
- * layout of the board and position of the keys.
- *
- * The second converts the arguments into a two-dimensional array which
- * represents the switch matrix.
- */
-
-// 2u backspace: XXX, k71
-// split backspace: k70, k71
+/*
+// 2u backspace: XXXX, k701
+// split backspace: k700, k701
 
-// 2u numpad +: k2c, XXX
-// split numpad +: k2c, k3c
+// 2u numpad +: k212, XXXX
+// split numpad +: k212, k312
 
-// 2u numpad enter: k4c, XXX
-// split numpad enter: k4c, k5c
+// 2u numpad enter: k412, XXXX
+// split numpad enter: k412, k512
 
-// 2.25u lshift: k40, XXX
-// split (1.25u, 1u) lshift: k40, k41
+// 2.25u lshift: k400, XXXX
+// split (1.25u, 1u) lshift: k400, k401
 
-// ANSI enter or ISO enter: k38
-// Key next to ISO enter: k72
-// backslash or NUHS: k73
+// ANSI enter or ISO enter: k308
+// backslash or NUHS: k208
 
-// left winkey: k50, k51, k52
-// left WKL: k50, XXX, k52
+// left winkey: k500, k501, k502
+// left WKL: k500, XXXX, k502
 
-// right winkey: k55, k56, k57
-// right WKL: k55, XXX, k57
+// right winkey: k505, k506, k507
+// right WKL: k505, XXXX, k507
+*/
 
 #define LAYOUT_all( \
-  k00,  k01, k02, k03, k04,  k60, k61, k62, k63,  k05, k06, k07, k08,   k72,  k09, k0a, k0b, k0c, \
-  k10, k11, k12, k13, k14, k64, k65, k66, k67, k15, k16, k17, k18, k70, k71,  k19, k1a, k1b, k1c, \
-  k20,  k21, k22, k23, k24, k68, k69, k6a, k6b, k25, k26, k27, k28,     k73,  k29, k2a, k2b, k2c, \
-  k30,    k31, k32, k33, k34, k6c, k75, k76, k77, k35, k36, k37,        k38,  k39, k3a, k3b, k3c, \
-  k40,  k41, k42, k43, k44, k78, k79, k7a, k7b, k45, k46, k47, k48,    k74,   k49, k4a, k4b, k4c, \
-  k50,  k51, k52,                k59,            k55, k56, k57,   k58, k53, k54,   k5a, k5b, k5c  \
+  k000, k001, k002, k003, k004, k600, k601, k602, k603, k005, k006, k007, k008,       k702,  k009, k010, k011, k012, \
```

Comment:
Please undo the changes to these layout macros, it makes it very difficult to see the actual changes to this file.